File: app/src/main/java/com/example/testingapplication/datamodel/ImageView.java
package com.example.testingapplication.datamodel;

import com.fasterxml.jackson.annotation.JsonProperty;


public class ImageView{
    @JsonProperty("_android:id")
    public String androidId;
    @JsonProperty("_android:layout_width")
    public String androidLayoutWidth;
    @JsonProperty("_android:layout_height")
    public String androidLayoutHeight;
    @JsonProperty("_app:srcCompat")
    public String appSrcCompat;
    @JsonProperty("_android:layout_x")
    public String androidLayoutX;
    @JsonProperty("_android:layout_y")
    public String androidLayoutY;
    @JsonProperty("_android:rotation")
    public String androidRotation;
    @JsonProperty("_android:background")
    public String androidBackground;
    @JsonProperty("_android:alpha")
    public String androidAlpha;
}

File: app/src/main/java/com/example/testingapplication/datamodel/ImageView.kt
package com.example.testingapplication.datamodel

import com.fasterxml.jackson.annotation.JsonProperty

class ImageView {
    @JsonProperty("_android:id")
    var androidId: String? = null

    @JsonProperty("_android:layout_width")
    var androidLayoutWidth: String? = null

    @JsonProperty("_android:layout_height")
    var androidLayoutHeight: String? = null

    @JsonProperty("_app:srcCompat")
    var appSrcCompat: String? = null

    @JsonProperty("_android:layout_x")
    var androidLayoutX: String? = null

    @JsonProperty("_android:layout_y")
    var androidLayoutY: String? = null

    @JsonProperty("_android:rotation")
    var androidRotation: String? = null

    @JsonProperty("_android:background")
    var androidBackground: String? = null

    @JsonProperty("_android:alpha")
    var androidAlpha: String? = null
}